roles/ipareplica/tasks/install.yml

```diff
@@ -102,6 +102,10 @@
       ipaclient_hostname: "{{ result_ipareplica_test.hostname }}"
       ipaclient_no_ntp: "{{ result_ipareplica_test.ipa_python_version
                             < 40690 }}"
+      ipaclient_ntp_servers:
+        "{{ omit if ipaclient_no_ntp else result_ipareplica_test.ntp_servers }}"
```

Comment:
`ipaclient_ntp_servers` will be set to `''` for the `ipaclient` role if `ipaclient_ntp_servers` was not set in the inventory. Please use something like `"{{ omit if ipaclient_no_ntp or result_ipareplica_test.ntp_servers == None else result_ipareplica_test.ntp_servers }}"`

Please also fix the other changes in the PR accordingly.

Reply:
I'm sorry, I couldn't follow your thinking.
Could you give me an example of an inventory that will cause this the failure you are suggesting so I can reproduce it locally?

Reply:
`result_ipareplica_test.ntp_servers` will be returned as `None` from `ipareplica_test` if `ipaclient_ntp_servers` is not set. As `ipaclient_ntp_servers` is a list, `"{{ result_ipareplica_test.ntp_servers }}"` will become "" and this is converted into `[""]`. This is not the same as `None`. Please have a look at 8ce5fd147aafc34e43dbe4246565c48eace2e115 where we had an issue because of this conversion. Therefore I want to make sure that we are not running into an issue now or later on. IPA code is always checking for `None`, but not for `[""]`. It will be needed to check the roles for more conversions like this one.